index.html

```diff
@@ -4509,154 +4509,129 @@ <h3>Usage Patterns</h3>
       </p>
     </section>
 
-    <section class="informative">
```

Comment:
~~~
<section class="informative" id="sharing-information-with-the-wrong-party">
~~~